Terraform/Home_Task_2/modules/asg/main.tf

```diff
@@ -42,73 +43,52 @@ resource "aws_autoscaling_group" "this_asg" {
   max_size             = 4
   desired_capacity     = 2

```

Comment:
min_size, max_size, desired_capacity - define values of these parameters in in general variables.tf file